lab-katherine/__test__/read.test.js

```diff
@@ -0,0 +1,42 @@
+'use-strict';
+const bark = require('../lib/read');
+const fs = require('fs');
+
+
+describe('My FS module', function() {
+  test('should return files in correct order', done => {
+    let first1, second2, third3;
+
+    fs.readFile(`${__dirname}/../assets/one.txt`, (err, data) => {
+      if (err) console.error(err);
+      first1 = data.toString('hex', 0, 8);
+    });
+
+    fs.readFile(`${__dirname}/../assets/two.txt`, (err, data) => {
+      if (err) console.error(err);
+      second2 = data.toString('hex', 0, 8);
+    });
+
+    fs.readFile(`${__dirname}/../assets/three.txt`, (err, data) => {
+      if (err) console.error(err);
+      third3 = data.toString('hex', 0, 8);
+    });
+
+    bark.readThing( (data) => {
+      let dataOG = [data.first, data.second, data.third];
+      let dataNew = [first1, second2, third3];
+      console.log(dataOG, dataNew);
+      expect(dataOG).toEqual(dataNew);
+      done();
+    });
+  });
+  test('should not recognize file path', done => {
+    let badPath;
+    fs.readFile(`${__dirname}/../assets/four.txt`, (err, data) => {
+      if(err) console.error(err);
+      badPath = data.toString('hex', 0, 8);
+    })
+    expect(badPath).toBeUndefined();
```

Comment:
This test is a false positive. Your not actually checking to see if bedPath is getting reassigned because the expect is happening outside of the callback. You will need to move the expect and done() into the callback to check if bedPath is receiving data.